migen/genlib/fifo.py

```diff
@@ -48,7 +52,8 @@ class _FIFOInterface:
         Acknowledge `dout`. If asserted, the next entry will be
         available on the next cycle (if `readable` is high then).
     """
-    def __init__(self, width, depth):
+
+    def __init__(self, width, depth, fwft=False):
```

Comment:
``FIFOInterface`` does not use ``fwft`` and therefore should not have this parameter.